src/webgl/shaders/line.vert

```diff
@@ -23,7 +23,7 @@ uniform mat4 uProjectionMatrix;
 uniform float uStrokeWeight;
 
 uniform vec4 uViewport;
-vec3 scale = vec3(1.0);
+vec3 scale = vec3(0.9995);
```

Comment:
great! worth a comment here about what this is doing (moving closer to camera), but otherwise this looks good to me.